src/gleam/set.gleam

```diff
@@ -330,7 +330,7 @@ pub fn is_disjoint(first: Set(member), from second: Set(member)) -> Bool {
 /// not both.
 ///
 /// ```gleam
-/// symmetric_difference(from_list([1, 2, 4]), from_list([3, 4])) |> to_list
+/// symmetric_difference(from_list([1, 2, 3]), from_list([3, 4])) |> to_list
 /// // -> [1, 2, 4]
```

Comment:
I double-checked this locally and now it matches what we have in the tests.

https://github.com/gleam-lang/stdlib/blob/fe517817bd4a58904e67fd462de817fccaaba92e/test/gleam/set_test.gleam#L133-L136